plenum/server/consensus/replica_service.py

```diff
@@ -16,7 +16,7 @@ class ReplicaService:
 
     def __init__(self, name: str, validators: List[str], primary_name: str,
                  timer: TimerService, bus: InternalBus, network: ExternalBus):
-        self._data = ConsensusSharedData(name, validators, 0)
+        self._data = ConsensusSharedData(name, 0)
```

Comment:
I suspect this can break simulation tests. Validators and primary name should be passed to ConsensusSharedData, either through constructor, or (if not available in constructor) by directly assigning properties, like `self._data.primary_name = primary_name`

Reply:
ok

Reply:
I see you now pass validators to ConsensusSharedData, but not primary name. You need to pass it here by calling
```
self._data.primary_name = primary_name
```

Reply:
ok, sure